Pickers/AbstractActionSheetPicker.m

```diff
@@ -779,6 +779,12 @@ - (void)presentPopover:(UIPopoverController *)popover {
 
 #pragma mark - Popoverdelegate
 
+- (BOOL)popoverControllerShouldDismissPopover:(UIPopoverController *)popoverController
+{
+    [self dismissPicker];
+    return YES;
```

Comment:
which case should it solve? I'm not sure, that we have to put in in abstreact class.